datasets/case01/queries/0010_leapSec.sql

```diff
@@ -0,0 +1,9 @@
+-- This query is frequently executed by buildbot
+
+-- See ticket #2048
+
+SELECT offset, mjdRef, drift, whenUtc
+FROM LeapSeconds
+WHERE whenUtc < 39900600000000000000000000
+ORDER BY whenUtc
```

Comment:
The ordering condition in the new query needs to be rewritten as 'ORDER BY whenUtc DESC' so that your 'LIMIT 1' would pick the maximum value of the timestamp as per the original query (with a subquery). That's because, according to the MySQL manual, the default sort order is ascending, with smallest values first. See:
https://dev.mysql.com/doc/refman/8.0/en/sorting-rows.html

Reply:
I did have that. Where did it go? ::grumble::

Reply:
(good catch)